inc/audiences/namespace.php

```diff
@@ -386,6 +340,15 @@ function admin_enqueue_scripts() {
 		'before'
 	);
 
+	// Only queue things up by default on the audience edit pages.
+	if ( get_current_screen()->id !== 'toplevel_page_' . POST_TYPE ) {
+		return;
+	}
+
+	wp_dequeue_script( 'post' );
+	wp_enqueue_script( 'altis-analytics-audience-ui' );
+
+	wp_enqueue_style( 'wp-components' );
```

Comment:
Just declare as a dep instead?

Reply:
Mmm, a style can't be a dep of a script.

Reply:
Right, but we're enqueuing the style immediately below.